app/src/test/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModelArrangement.kt

```diff
@@ -225,5 +224,27 @@ internal class NewConversationViewModelArrangement {
         coEvery { isSelfTeamMember() } returns result
     }
 
+    fun withGuestEnabled(x: Boolean) = apply {
```

Comment:
just a tiny readability suggestion
```suggestion
    fun withGuestEnabled(isGuestModeEnabled: Boolean) = apply {
```